schema.sql

```diff
@@ -9,3 +9,7 @@ CREATE TABLE animals (
     weight_kg DECIMAL NOT NULL,
     PRIMARY KEY(id),
 );
```

Comment:
- [ ] Please remove comma at the end of line 10 because it is causing an syntax error :fearful:

Reply:
done